eth-tests/src/main/java/tech/pegasys/artemis/ethtests/TestSuite.java

```diff
@@ -159,28 +160,26 @@ public static Object pathToObject(Path path, List<TestObject> testObjects) {
                           testSet.getFileNames().stream()
                               .flatMap(
-                                  fileName -> {
-                                    Object object =
-                                        pathToObject(
-                                            Path.of(walkPath, fileName),
-                                            testSet.getTestObjectByFileName(fileName));
-                                    return testSet.getTestObjectByFileName(fileName).stream()
-                                        .map(
-                                            testObject -> {
-                                              if (fileName.contains(".ssz")) {
-                                                Bytes objectBytes =
-                                                    getSSZBytesFromPath(
-                                                        Path.of(walkPath, fileName));
-                                                return SimpleOffsetSerializer.deserialize(
-                                                    objectBytes, testObject.getClassName());
-                                              } else {
-                                                return parseObjectFromFile(
-                                                    testObject.getClassName(),
-                                                    testObject.getPath(),
-                                                    object);
-                                              }
-                                            });
-                                  }),
+                                  fileName ->
+                                      testSet.getTestObjectByFileName(fileName).stream()
+                                          .map(
+                                              testObject -> {
+                                                if (fileName.contains(".ssz")) {
+                                                  Bytes objectBytes =
+                                                      getSSZBytesFromPath(
+                                                          Path.of(walkPath, fileName));
+                                                  return SimpleOffsetSerializer.deserialize(
+                                                      objectBytes, testObject.getClassName());
+                                                } else {
+                                                  return parseObjectFromFile(
+                                                      testObject.getClassName(),
+                                                      testObject.getPath(),
+                                                      pathToObject(
+                                                          Path.of(walkPath, fileName),
+                                                          testSet.getTestObjectByFileName(
+                                                              fileName)));
```

Comment:
The only difference is that this `pathToObject` call was inlined so it only executes if it's not a `.ssz` file (which would fail and log a warning).